config/config.go

```diff
@@ -29,6 +28,9 @@ type (
 
 		// AsyncService is config for async service
 		AsyncService *AsyncServiceConfig `yaml:"asyncService"`
+
+		// Membership is the config for cluster members
+		Membership *MembershipConfig `yaml:"membership"`
```

Comment:
Now MembershipConfig is in the same level as ApiServiceConfig and AsyncServiceConfig.

Reply:
Nice!